Form-Controls/styles.css

```diff
@@ -0,0 +1,74 @@
+
+body{
+    background: #F68E5F;  
+    text-align: center;
+    width: 30%;
+    margin:auto;
+}
+main{
+    display: flex;
+    flex-direction: column;
+    align-items:center  ;
+    justify-content: center;
+    background: #F5DD90;
+    border-radius: 20px;
+    padding-bottom: 20px;
+}
+
+#heading{
+    margin: 0 0 50px 0;
+}
+
+input{
+    width: 100%;
+    padding: 12px 20px;
+    margin: 8px 0;
+    box-sizing: border-box;
+    background-color: white;
+    color: black;
+  }
+
+
+#name, #mail, #date{
```

Comment:
Limit your use of pixel sizing and limit your use of explicit heights when sizing text containers in CSS.